src/photon/photon_client.c

```diff
@@ -15,6 +15,28 @@ void photon_disconnect(photon_conn *conn) {
   free(conn);
 }
 
+void photon_log_event(photon_conn *conn,
+                      uint8_t *key_name,
+                      int64_t key_name_length,
+                      uint8_t *payload,
+                      int64_t payload_length) {
+  int64_t message_length = sizeof(key_name_length) + sizeof(payload_length) +
```

Comment:
let's put a TODO here that this should be flatbufferized (then we can also pass the flatbuffers through from python directly)